api/overview/azure/powerbi-embedded.md

```diff
@@ -15,25 +15,64 @@ ms.service: multiple
 
 # Azure PowerBI Embedded libraries for .NET

```

Comment:
If we can drop "Azure" from the title, I would prefer that. I think that may be a requirement though. The actual product name is just Power BI Embedded. Not Azure Power BI Embedded.